mobile_packet_verifier/README.md

```diff
@@ -0,0 +1,21 @@
+# Mobile Packet Verifier
+
+The mobile packet verifier reads data transfer sessions and accumulates the amount
+of bytes downloaded and uploaded. After a specified period, it burns a proportional
+amount of data credits from the payer and issues validated data transfer sessions 
+so that the mobile verifier may reward the hotspots. 
+
+The mobile packet verifier assumes that all payers have sufficient balance.
```

Comment:
Is this fair to say? Doesn’t it make this assumption to within a certain buffer and then starts rejecting packets if it’s not able to perform a successful burn?

Reply:
I guess it technically doesn't _assume_, let me re-word this

Reply:
elaborated on this a bit